src/index.ts

```diff
@@ -0,0 +1,125 @@
+/**
+ * @license
+ * Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+ */
+'use strict';
+
+import {Logger, LoggerInstance, ConsoleTransportInstance, transports} from 'winston';
+
+class PolymerLogger {
+
+  _transport: ConsoleTransportInstance;
+  _logger: LoggerInstance;
+
+  /**
+   * Constructs a new instance of PolymerLogger. This creates a new internal
+   * `winston` logger, which is what we use to handle most of our logging logic.
+   */
+  constructor(options?: {level?: string, name?: string}) {
+    options = options || {};
+
+    this._transport = new (transports.Console)({
```

Comment:
Unnecessary parens, `new transports.Console(/* ... */);` is equivalent and easier to read